bode/bode/resources/tasks/schemas.py

```diff
@@ -23,3 +23,4 @@ class TaskSchema(BaseSchema):
     due_date = fields.DateTime()
     status = fields.String()
     tags = fields.List(fields.Nested(TagSchema))
+    relation_types = fields.List(fields.String())
```

Comment:
```suggestion
    relation_types = fields.List(fields.String(validate=validate.OneOf(DirectedRelationType.list())))
```

You can use validate to mention that is Enum type and to be consistent with other schemas ([RelatedTaskSchema](bode/bode/resources/task_relations/schemas.py)).